components/gc-servinfo/gc-srvinfo-examples.html

```diff
@@ -2,10 +2,9 @@
 {
 	"title": "Services and information - Working examples",
 	"language": "en",
-	"altLangPrefix": "gc-srvinfo-examples",
-	"breadcrumb": [
+	"breadcrumbs": [
 		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html" },
-		{ "title": "gc-srvinfo", "link": "gc-srvinfo.html" }
+		{ "title": "gc-srvinfo", "link": "components/gc-servinfo/gc-srvinfo.html" }
```

Comment:
FWIW this and everything else like it could  be changed back to the old link format (i.e. relative to the page instead of relative to ``site.baseurl``) by removing Liquid's ``relative_url`` filter from ``sites/breadcrumbs/inc-breadcrumbs.html``.